src/ui/components/ConfirmButton/index.js

```diff
@@ -116,11 +117,18 @@ export class ConfirmButtonBase extends React.Component<InternalProps> {
   }
 }
 
+export const extractId = (ownProps: Props) => {
+  const buttonId =
+    ownProps.id ||
+    `${ownProps.className || ''}-${ownProps.message.replace(/\s/g, '')}`;
+  return `confirmButton-${buttonId}`;
```

Comment:
prefixing with `confirmButton` is redundant since it will already be prefixed with the filename